app/services/activitypub/fetch_remote_account_service.rb

```diff
@@ -5,8 +5,13 @@ class ActivityPub::FetchRemoteAccountService < BaseService
 
   # Should be called when uri has already been checked for locality
   # Does a WebFinger roundtrip on each call
-  def call(uri, prefetched_json = nil)
-    @json = body_to_json(prefetched_json) || fetch_resource(uri)
+  def call(uri, resource_option)
```

Comment:
I don't like the overloading of the 2nd argument here, maybe `options = {}` (or named arguments) would be better

Reply:
I think that's my main comment, because of how this is used in calls it's hard to understand what is happening.